.github/workflows/release-internal.yml

```diff
@@ -28,7 +40,7 @@ jobs:
   invoke-reusable-workflow:
     name: "Build and Release Internally"
 
-    uses: "VersusFacit/dbt-release/.github/workflows/internal-archive-release.yml@main"
+    uses: "dbt-labs/dbt-release/.github/workflows/internal-archive-release.yml@mp/finish_internal_workflow"
```

Comment:
Going to change this once `main` has all the changes needed to support all the other workflows.